README.md

```diff
@@ -26,12 +26,16 @@ Virtual Environment setup:
 Running møte:
  - Start `memcached` by typing `memcached` in the terminal and pressing Return (Enter) key.
  - Edit [config.py](./files/config.py) with necessary information and place the `config.py` in [`mote/`](./mote)
- - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE`
+ - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE` (Optional)
```

Comment:
We wouldn't need `memcached` from now on, as It's by default `FALSE` and It'll default to file based cache.

Reply:
Ah, a README.md change. I'd make this one real quick.